languages/csharp/exercises/concept/chars/.meta/design.md

```diff
@@ -0,0 +1,56 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of Chars in [C#][chars-docs].
+
```

Comment:
We've recently removed the `## Goal` section from the `design.md` document, as it didn't add any value.